### DISCUSSION TITLE
Reason for use of intl-locales-supported version that warns "bad publish"?

### DISCUSSION THREAD

**Discussion by jameswjr** [facebook/docusaurus#4435] [Q&A]
I recently installed the 2.0.0-alpha.71 version, and got a warning of "bad publish" from yarn for the version used of "intl-locales-supported", which is 1.8.12 and which is apparently deprecated.  This appears to be a new dependency for this version of docusaurus-utils, so this is a fresh problem.  Is there a reason this version was used in docusaurus-utils, vs. the prior version which has no such warning and is not deprecated?  And what is a "bad publish" and what does it mean for the integrity of the installation of the package and of docusuaurus?

**Comment by slorber** [comment]
Already fixed by https://github.com/facebook/docusaurus/pull/4384 and may be removed by https://github.com/facebook/docusaurus/pull/4405
